README.md

```diff
@@ -587,6 +588,26 @@ migration "state" "test" {
 }
 ```
 
+#### state xmv
+
+The `xmv` command works like the `mv` command but allows usage of
+wildcards `*` in the source definition. The source expressions will be 
+matched against resources defined in the terraform state. The matched value
+can be used in the destination definition via a dollar sign and their ordinal number: 
+`$1`, `$2`, ... When there is ambiguity the ordinal number can be put in curly braces (e.g. `${1}`).
+
+For example if `foo` and `bar` in de `mv` command example above are the only 2 security group resources
```

Comment:
Is `de` a typo?

Reply:
Yes, it's dutch for `the`, I will correct it.